docs/source/deploy_chaincode.md

```diff
@@ -0,0 +1,348 @@
+
+## Package the smart contract
+
+We need to package the chaincode before it can be installed on our peers. The steps are different if you want to install a smart contract written in [Golang](#golang), [Java](#java), or [JavaScript](#javascript).
+
+### Golang
+
+Before we package the chaincode, we need to install the chaincode dependences. Navigate to the folder that contains the go version of the Fabcar chaincode.
+```
+cd fabric-samples/chaincode/fabcar/go
+```
+
+The sample uses a go module to install the chaincode dependencies. The dependencies are listed in a `go.mod` file in the `fabcar/go` directory. You should take a moment to examine this file.
+```
+$ cat go.mod
+module github.com/hyperledger/fabric-samples/chaincode/fabcar/go
+
+go 1.13
+
+require github.com/hyperledger/fabric-contract-api-go v0.0.0-20191118113407-4c6ff12b4f96
+```
+The `go.mod` file imports the Fabric contract API into the smart contract package. You can open `fabcar.go` in a text editor to see how the contract API is used to define the `SmartContract` type at the beginning of the smart contract:
+```
+// SmartContract provides functions for managing a car
+type SmartContract struct {
+	contractapi.Contract
+}
+```
+
+The ``SmartContract`` type is then used to create the transaction context for the functions defined within the smart contract that read and write data to the blockchain ledger.
+```
+// CreateCar adds a new car to the world state with given details
+func (s *SmartContract) CreateCar(ctx contractapi.TransactionContextInterface, carNumber string, make string, model string, colour string, owner string) error {
+	car := Car{
+		Make:   make,
+		Model:  model,
+		Colour: colour,
+		Owner:  owner,
+	}
+
+	carAsBytes, _ := json.Marshal(car)
+
+	return ctx.GetStub().PutState(carNumber, carAsBytes)
+}
+```
+You can learn more about the Go contract API by visiting the [API documentation](https://github.com/hyperledger/fabric-contract-api-go) and the [smart contract processing topic](developapps/smartcontract.html).
+
+To install the smart contract dependencies, run the following command from the `fabcar/go` directory.
+```
+GO111MODULE=on go mod vendor
+```
+
+If the command is successful, the go packages will be installed inside a `vendor` folder.
+
+Now that we that have our dependences, we can create the chaincode package. Navigate back to our working directory in the `test-network` folder so that we can package the chaincode together with our other network artifacts.
+```
+cd ../../../test-network
+```
+
+You can use the `peer` CLI to create a chaincode package in the required format. The `peer` binaries are located in the `bin` folder of the `fabric-samples` repository. Use the following command to add those binaries to your CLI Path:
+```
+export PATH=${PWD}/../bin:${PWD}:$PATH
+```
+You also need to set the `FABRIC_CFG_PATH` to point to the `core.yaml` file in the `fabric-samples` repository:
+```
```

Comment:
Can you briefly explain why the core.yaml is needed for this process?

Reply:
I do not know and I assume it is a bug.